pkg/convert/helpers.go

```diff
@@ -0,0 +1,11 @@
+package convert
+
+// windowStateToIsOpen converts the Tesla WindowState enum, which we typically receive
+// as a string, to 1 (open) or 0 (closed).
+// See https://github.com/teslamotors/fleet-telemetry/blob/646fce2fb2ddd607ce4e76c865ce411e32ded81f/protos/vehicle_data.proto#L465
+func windowStateToIsOpen(state string) float64 {
+	if state == "PartiallyOpen" || state == "Opened" {
```

Comment:
Maybe we should error if we get something unexpected.

Reply:
Yeah I hate that I can't trust that a new random state won't be added. Maybe if we flip this where closed is 0 and everything else is 1.

Reply:
I'm happy to... I may error. Let's error.